IM21DoorStatusEnum.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const IM21_DOOR_STATE = {
```

Comment:
Thanks for making this a constant! :smile:  This is makes everything else much more readable.